stdlib/public/SwiftShims/LibcShims.h

```diff
@@ -93,22 +93,38 @@ _swift_stdlib_cxx11_mt19937_uniform(__swift_uint32_t upper_bound);
 // Math library functions
 static inline SWIFT_ALWAYS_INLINE
 float _swift_stdlib_remainderf(float _self, float _other) {
+#if __has_builtin(__builtin_remainderf)
   return __builtin_remainderf(_self, _other);
+#else
+  return remainderf(_self, _other);
```

Comment:
If this header is pulled in only transitively, could the floating point shims simply be moved out into a separate FloatingPointShims.h?